assignments/array-methods.js

```diff
@@ -56,28 +56,49 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach( function(name) {
+    fullName.push(`${name.first_name} ${name.last_name}`);
+
+});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+allCaps = runners.map(x => x.first_name.toUpperCase())
 console.log(allCaps); 
```

Comment:
I like the fact that you used x to make the function more concise but using 'runner' would be more semantic and make the code more readable.